style.css

```diff
@@ -1198,11 +1262,154 @@ textarea::placeholder {
     box-sizing: border-box;
     border: 1px solid #dfe1e6;
     align-content: center;
+    margin-left: 40px;
   }
 
   .card {
     display: flex;
     justify-content: flex-start;
     padding: 0;
   }
+
+  .close {
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    backdrop-filter: blur(6px);
+    align-items: start;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 32px;
+    line-height: 44px;
+    margin: 15px;
+    margin-right: 10px;
+    margin-left: 70px;
+    margin-bottom: 20px;
+    border: 1px solid black;
+    width: 80vw;
+    min-height: max-content;
+    border-radius: 12px;
+    padding: 16px;
+    zoom: 0;
+  }
+
+  .work {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.2vw;
+    line-height: 1.9vw;
+    color: #344563;
+    padding-bottom: 20px;
+    width: 54vw;
+  }
+
+  .work-lang {
+    display: flex;
+    flex-direction: row;
+    padding: 25px;
+  }
+
+  .detailsw {
+    font-style: normal;
+    font-size: 1em;
+    display: flex;
+    align-items: center;
+    padding-left: 0;
+    gap: 3%;
+
+    /* margin-top: 16px;
+  margin-bottom: 16px; */

```

Comment:
- [ ] It is considered best practice to avoid keeping commented codes or unused codes in your work base, as it makes it messy. Kindly remove this unused part of your code. :wrench: